nifi-toolkit/nifi-toolkit-admin/src/test/groovy/org/apache/nifi/toolkit/admin/client/NiFiClientFactorySpec.groovy

```diff
@@ -71,6 +93,71 @@ class NiFiClientFactorySpec extends Specification {
 
     }
 
+    def "should verify CN in certificate based on subjectDN"(){
```

Comment:
I like these tests. If you add a test for the manipulated DN I posted above, you can verify that the `LdapName` implementation works with that.